include/zephyr/net/net_event.h

```diff
@@ -257,7 +256,19 @@ struct net_event_ipv6_route {
 	uint8_t prefix_len;
 };
 
-#endif /* CONFIG_NET_MGMT_EVENT_INFO */
```

Comment:
Looks that the documentation behind ifdef was not parsed by doxygen before:

> WARNING: /runner/_work/zephyr/zephyr/include/zephyr/net/net_event.h:238: Found unknown command '`@Note'`

Should be `@note`, can you please fix this.